install.sh

```diff
@@ -393,6 +393,32 @@ set_current_context_k8s() {
   fi
 }
 
+scale_down() {
+    set +e
+    PRODUCTS=$(grep -o '^[^#]*' "${CONFIG_ABS_PATH}" | grep "products" | sed 's/ //g')
+    PRODUCTS="${PRODUCTS#*=}"
+    PRODUCTS_ARRAY=($(echo $PRODUCTS | sed 's/\[//g' | sed 's/\]//g' | sed 's/,/ /g' | sed 's/"//g'))
+    if echo "$PRODUCTS" | grep -qE 'jira|confluence'; then
+      SNAPSHOTS_JSON_FILE_PATH=$(get_variable 'snapshots_json_file_path' "${CONFIG_ABS_PATH}")
+      if [ "${PATH}" ]; then
+        local EKS_PREFIX="atlas-"
```

Comment:
Are these fixed constants? Any chance we can refer to them rather than magical string?